sdks/python/apache_beam/runners/interactive/display/pipeline_graph.py

```diff
@@ -182,8 +182,8 @@ def _generate_graph_dicts(self):
       for pcoll_id in transform.outputs.values():
         pcoll_node = None
         if self._pipeline_instrument:
-          pcoll_node = self._pipeline_instrument.cacheable_var_by_pcoll_id(
-              pcoll_id)
+          cacheable = self._pipeline_instrument._cacheables.get(pcoll_id)
```

Comment:
It's a code smell when another class access the private members of another. Is there another way to do this?

Reply:
We can expose the cacheables as the property of a PipelineInstrument.

I removed the property because I don't want PipelineInstrument to be abused by other modules. In most cases, we should directly get cacheables through the utility/inspector, not through building a PipelineInstrument.